packages/flutter/test/material/text_field_test.dart

```diff
@@ -3382,4 +3382,35 @@ void main() {
     await tester.tap(find.byType(TextField));
     expect(tapCount, 0);
   });
+
+  testWidgets('style enforces required fields', (WidgetTester tester) async {
+    Widget buildFrame(TextStyle style) {
+      return MaterialApp(
+        home: Material(
+          child: TextField(
+            style: style,
+          ),
+        ),
+      );
+    }
+
+    await tester.pumpWidget(buildFrame(const TextStyle(
```

Comment:
I think what you want here is:
````
expect(buildFrame(...), throwsAssertionError);
```

Reply:
Talked to Hans and it seems we can't do it this way because `build` will never be called.